content/gsoc-2025-selected-projects.md

```diff
@@ -0,0 +1,42 @@
++++
+path = "2025/05/08/gsoc-2025-selected-projects"
+title = "Announcing Google Summer of Code 2025 selected projects"
+authors = ["Jakub Beránek"]
+
+[extra]
+team = "the mentorship team"
+team_url = "https://www.rust-lang.org/governance/teams/launching-pad#team-mentorship"
++++
+
+The Rust Project is [participating][gsoc blog post] in [Google Summer of Code (GSoC) 2025][gsoc] again this year. GSoC is a global program organized by Google that is designed to bring new contributors to the world of open-source.
+
+In March, we published a list of [GSoC project ideas][project idea list], and started discussing these projects with potential GSoC applicants on our [Zulip][zulip gsoc]. We had many interesting discussions with the potential contributors, and even saw some of them making non-trivial contributions to various Rust Project repositories, even before GSoC officially started!
+
+After the initial discussions, GSoC applicants prepared and submitted their project proposals. We received 64 proposals this year, almost exactly the same number as last year. We are happy to see that there was again so much interest in our projects.
+
+A team of mentors primarily composed of Rust Project contributors then thoroughly examined the submitted proposals. GSoC required us to produce a ranked list of the best proposals, which was a challenging task in itself since Rust is a big project with many priorities! Same as last year, we went through several rounds of discussions and considered many factors, such as prior conversations with the given applicant, the quality of their proposal, the importance of the proposed project for the Rust Project and its wider community, but also the availability of mentors, who are often volunteers and thus have limited time available for mentoring.
+
+As is usual in GSoC, even though some project topics received multiple proposals[^most-popular], we had to pick only one proposal per project topic. We also had to choose between great proposals targeting different work to avoid overloading a single mentor with multiple projects.
+
+[^most-popular]: The most popular project topic received seven different proposals!
+
+In the end, we narrowed the list down to TODO best proposals, which we felt was the maximum amount that we could realistically support with our available mentor pool. We submitted this list and eagerly awaited how many of these TODO proposals would be accepted into GSoC.
```

Comment:
As mentioned on zulip, I agree we probably shouldn't mention # of proposals we narrowed to.

Reply:
Agreed, I already removed it in the final version.